src/components/SearchResults.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from "react";
+import moment from "moment";
+
+export default class SearchResults extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  rowSelected = event => {
+    this.setState(pre => {
+      return { selected: !pre.selected };
+    });
+
+    this.state.selected
```

Comment:
You don't want to directly modify the DOM in this way. Ali will explain why tomorrow in class.

Instead, modify your component's state, and in the render method, build the DOM according to the state.